### DISCUSSION TITLE
Automatically Adjust Font Metrics to Match a Different Font

### DISCUSSION THREAD

**Discussion by Gwrgwer-Ofgqfrw** [fonttools/fonttools#3148] [General]
I have two fonts:

- **font A**: open-source font
- **font B**: proprietary font whose metrics I'm trying to match.

How do I create a replacement font (**font C**) with the glyphs from  **font A** but with the metrics from font B?

Also, I want all characters not present in **font B** not to be present in **font C** and I want to receive a warning for every character present in **font B** but not in  **font A**, so that I can create a replacement that supports the exact same characters.

There is already another discussion from another user asking this, but there is no answer so I decided to ask a new one.

Thank you a lot @justvanrossum @anthrotype @behdad @brawer for your fantastic contributions to this repo.

**Comment by justvanrossum** [comment]
Why do you keep making new github accounts? Why do you spam other topics?

**Reply by anthrotype** [reply]
I just blocked them but they'll continue to create new GitHub profiles and there's not much we can do about it I'm afraid
